apps/condo/domains/ticket/utils/testSchema/index.js

```diff
@@ -820,6 +822,37 @@ async function updateTestIncidentClassifierIncident (client, id, extraAttrs = {}
     return [obj, attrs]
 }
 
+async function createTestUserFavoriteTicket (client, user, ticket, extraAttrs = {}) {
```

Comment:
createTestUserFavo**U**riteTicket

Reply:
favoUrite and favorite - both words are correct